gridstatus/pjm.py

```diff
@@ -941,6 +941,58 @@ def extract_short_name(row):
 
         return data
 
+    @support_date_range(frequency=None)
+    def get_it_sced_lmp_5_min(self, date, end=None, verbose=False):
+        """Get 5 minute LMPs from the Integrated Forward Market (IFM)"""
+
+        if date == "latest":
+            return self.get_it_sced_lmp_5_min("today", verbose=verbose)
+
+        params = {
+            "fields": (
+                "case_approval_datetime_utc,datetime_beginning_utc,itsced_lmp,marginal_congestion,marginal_loss,pnode_id,pnode_name"  # noqa: E501
+            ),
+        }
+
+        data = self._get_pjm_json(
+            "five_min_itsced_lmps",
+            start=date,
+            end=end,
+            params=params,
+            verbose=verbose,
+            interval_duration_min=5,
+        )
+
+        # TODO: are there other columns we want to rename?
+        data = data.rename(
+            columns={
+                "case_approval_datetime_utc": "Case Approval Time",
+            },
+        )
+
+        data.columns = data.columns.map(lambda x: x.replace("_", " ").title())
+
+        data = data[
+            [
+                "Interval Start",
+                "Interval End",
+                "Case Approval Time",
+                "Pnode Id",
+                "Pnode Name",
+                "Itsced Lmp",
```

Comment:
for consistency with the other pjm lmp dataset, i think we should calculate and add in the `marginal energy`

https://github.com/gridstatus/gridstatus/blob/44aa6111e21a3a99c58a7df83154a1af7b7399da/gridstatus/pjm.py#L863-L864